### PR TITLE
changes structure for original and corrected test statistics

### PR DESCRIPTION
**Pull request by daryadedik**
OriginalTestStatistics now holds original_test_statistics (of type SimpleTestStatistics or EarlyStoppingTestStatistics), OriginalAndCorrectedTestStatistics holds original_test_statistics and corrected_test_statistics (same types).

### CODE REVIEW

**Comment by shansfolder** [comment]
Thanks Dasha! I will add the docs in the same branch..

**Comment by daryadedik** [comment]
not finished yet, do not review yet.

File: expan/core/results.py [hunk 1]
```diff
@@ -76,8 +76,19 @@ def __init__(self, control_statistics, treatment_statistics, delta, ci, p, stati
...
+    """ Holds original statistics without corrected statistics.
+    
+    :param original_test_statistics: test result without correction
+    :type  original_test_statistics: SimpleTestStatistics or EarlyStoppingTestStatistics
+    """
+    def __init__(self, original_test_statistics):
+        self.original_test_statistics = original_test_statistics
+
+
+class OriginalAndCorrectedTestStatistics(OriginalTestStatistics):
```
Comment (gbordyugov) [inline expan/core/results.py:89]: this inheritance looks strange to me

File: expan/core/results.py [hunk 2]
```diff
@@ -76,8 +76,19 @@ def __init__(self, control_statistics, treatment_statistics, delta, ci, p, stati
...
-class CorrectedTestStatistics(JsonSerializable):
-    """ Holds original and corrected statistics. This class should be used to hold statistics for multiple testing.
+class OriginalTestStatistics(JsonSerializable):
+    """ Holds original statistics without corrected statistics.
+    
+    :param original_test_statistics: test result without correction
+    :type  original_test_statistics: SimpleTestStatistics or EarlyStoppingTestStatistics
+    """
+    def __init__(self, original_test_statistics):
+        self.original_test_statistics = original_test_statistics
```
Comment (gbordyugov) [inline expan/core/results.py:86]: What's the exact point of this class if it only holds one field?

**Pull request closed without merging**